forge/forge/llm/providers/multi.py

```diff
@@ -69,7 +69,11 @@ async def get_available_models(self) -> Sequence[ChatModelInfo[ModelName]]:
     async def get_available_chat_models(self) -> Sequence[ChatModelInfo[ModelName]]:
         models = []
         for provider in self.get_available_providers():
-            models.extend(await provider.get_available_chat_models())
+            if hasattr(provider, 'is_configured') and provider.is_configured():
+                try:
+                    models.extend(await provider.get_available_chat_models())
+                except Exception as e:
+                    logger.warning(f"Failed to get models from {provider.__class__.__name__}: {e}")
```

Comment:
This isn't going to fix the issue:
1. It seems there's no `is_configured` in any of the providers, so no models will be added.
2. `get_available_chat_models()` just lists model names from a constant and has nothing to do with provider being ready or not.

`create_chat_completion` calls `get_model_provider` and then `_get_provider` which triggers initialization if needed.